cfgov/paying_for_college/disclosures/scripts/update_colleges.py

```diff
@@ -36,7 +37,12 @@ def fix_zip5(zip5):
 
 
 def get_scorecard_data(url):
-    response = requests.get(url)
+    """Make our API call to Scorecard."""
+    try:
+        response = requests.get(url)
+    except SSLError:
+        logger.warn("SSL error connecting with Scorecard")
```

Comment:
Might want to use [`logger.exception`](https://docs.python.org/3/library/logging.html#logging.Logger.exception) so that you dump out the exception details. Out of curiosity, when have you seen this happen?

Reply:
Only a few sporadic times recently. It's a pain if it's not caught, because then you lose reporting on closed schools up to that point.

Reply:
√